### PR TITLE
Integrated auth and draft hackathon page added

### PR DESCRIPTION
**Pull request by subodhk01**

### CODE REVIEW

File: util/constants.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export const API_URL = "http://127.0.0.1:8000/"
```
Comment (nishantwrp) [inline util/constants.js:1]: Why not use cops-hackalog.herokuapp.com even while development?

Comment (subodhk01) [reply util/constants.js:1]: Ohh yeah, right!

Comment (subodhk01) [reply util/constants.js]: Done

File: .gitignore [hunk 2]
```diff
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+package-lock.json
```
Comment (nishantwrp) [inline .gitignore:7]: Why this?

Comment (subodhk01) [reply .gitignore:7]: Koi sense hai remote pe rakhne ka ise?

Comment (nishantwrp) [reply .gitignore:7]: Time km lgega naa dependencies install krne ka fir

Comment (subodhk01) [reply .gitignore]: Fixed

**Review by nishantwrp — APPROVED** [review]

**Pull request merged**